packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton.tsx

```diff
@@ -43,15 +38,21 @@ export default function RainbowKitCustomConnectButton() {
               }
 
               if (chain.unsupported || chain.id !== configuredChain.id) {
-                return (
-                  <div className="rounded-md shadow-lg p-2">
-                    <span className="text-error mr-2">Wrong network selected - ({chain.name})</span>
-                    <span className="text-primary mr-2">Switch network to</span>
-                    <button className="btn btn-xs btn-primary btn-outline" onClick={onSwitchNetwork}>
-                      {configuredChain.name}
-                    </button>
-                  </div>
-                );
+                if (chain.id === 1) {
+                  return (
+                    <div className="rounded-md shadow-lg p-2">
+                      <button
+                        className="text-white flex items-center gap-1 rounded-[10px] bg-red-500 px-4 py-2"
```

Comment:
The button has a weird border around it.

![image](https://user-images.githubusercontent.com/2486142/220920581-57fc7468-4215-4d0a-881e-b4ae1a1412a9.png)

Reply:
Also we should use daisy colors defined on `tailwind.config.js` and the `btn` classes that we are using in other buttons

Reply:
ok i will take that off

Reply:
which class contains the specific color. i have checked btn-error and btn-warning, the colors are different